hr_commission/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2015-2020 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Tecnativa - João Marques
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "HR commissions",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/commission",
+    "category": "Commissions",
+    "depends": ["sale_commission", "hr"],
```

Comment:
No need of depending on the sales one now.

```suggestion
    "depends": ["account_commission", "hr"],
```